### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by brunojdo**
#### What is the purpose of this pull request?

As the title says 
#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,7 +24,8 @@ See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for mor
 - [Troubleshooting](#troubleshooting)
 
 ## Usage
-:construction: :construction: :construction:
+
+This app use our store builder with the blocks architecture. To know more about Store Builder [click here.](https://help.vtex.com/en/tutorial/understanding-storebuilder-and-stylesbuilder#structuring-and-configuring-our-store-with-object-object)
```
Comment (vitoria) [inline README.md]: :hammer: This app `uses`...

File: README.md [hunk 2]
```diff
@@ -24,7 +24,8 @@ See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for mor
...
 
 To use this app you need to import in your dependencies on `manifest.json`.
```
Comment (vitoria) [inline README.md]: :hammer:  you need to import `it`...

File: README.md [hunk 3]
```diff
@@ -34,14 +35,26 @@ To use this app you need to import in your dependencies on `manifest.json`.
...
 ```
 
-## API 
-:construction: :construction: :construction:
+Then, add `minicart` block into our app theme, like we do in our [Dreamstore app](https://github.com/vtex-apps/dreamstore/blob/master/store/blocks.json). 
 
 ### Blocks API
 :construction: :construction: :construction:
 
+This app have an interface that describes what are the rules that the blocks must implement when you want to use the minicart block.
```
Comment (vitoria) [inline README.md]: :hammer:  This app `has` ...

File: README.md [hunk 4]
```diff
@@ -34,14 +35,26 @@ To use this app you need to import in your dependencies on `manifest.json`.
...
+```json
+  "minicart": {
+    "required": [
+      "product-summary"
+    ],
+    "component": "index"
+  }
+}
+```
+The minicart has as a required block the `product-summary` block. So minicart implementation that will be created must add a product-summary as block that is inside of minicart. To know how is the API of `product-summary` see the next section.
```
Comment (vitoria) [inline README.md]: :hammer:  ... to know how the `product-summary` API is, see the next section...

**Review by vitoria — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (theomoura) [inline README.md]: The minicart has as a required block, the `product-summary` one 🔨

**Review by theomoura — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (GustavoMA) [reply README.md]: @theomoura isn't better?
The minicart has a required block that is `product-summary`.

File: README.md [hunk 5]
```diff
@@ -34,14 +35,26 @@ To use this app you need to import in your dependencies on `manifest.json`.
   }
 ```
 
-## API
```
Comment (GustavoMA) [inline README.md:37]: Fix `Table of Contents` since you removed API.

File: README.md [hunk 6]
```diff
@@ -2,8 +2,7 @@ VTEX Minicart
 =====
 
 ## Description
-The VTEX minicart app shows a list of all items that a customer add in our Checkout OrderForm API. 
-This is a VTEX app that is used by Dreamstore product.
+The VTEX minicart app shows a list of all items that a customer add in our Checkout OrderForm API, and this app is used by Dreamstore product.
```
Comment (vitoria) [inline README.md]: :hammer:  ... that a customer `added` in our ...

Comment (GustavoMA) [inline README.md]: ` Checkout OrderForm API and this` :hammer: 
No need for the `,`

File: README.md [hunk 7]
```diff
@@ -17,31 +16,43 @@ See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for mor
...
 ```
 
-## API 
-:construction: :construction: :construction:
+Then, add `minicart` block into our app theme, as we do in our [Dreamstore app](https://github.com/vtex-apps/dreamstore/blob/master/store/blocks.json). 
 
 ### Blocks API
 :construction: :construction: :construction:
 
+This app has an interface that describes what the rules that must a block implement when you want to use the minicart.
```
Comment (vitoria) [inline README.md]: I think that this text would be better this way:

```
This app has an interface that describes what the rules that must be implemented by a block when you want to use the minicart. 
```

**Review by GustavoMA — CHANGES_REQUESTED** [review]

**Review by theomoura — APPROVED** [review]

File: README.md (hunk 7 above)
Comment (paul0vinicius) [reply README.md]: `This app has an interface that describes what rules must be implemented by a block when you want to use the minicart. `

What do you think?

File: README.md [hunk 8]
```diff
@@ -17,31 +16,43 @@ See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for mor
...
+```json
+  "minicart": {
+    "required": [
+      "product-summary"
+    ],
+    "component": "index"
+  }
+}
+```
+The minicart has as a required block the `product-summary`. So, any minicart implementation created must add a product-summary as a block that is inside of minicart. To know how is the API of `product-summary` see the next section.
```
Comment (vitoria) [inline README.md:52]: :hammer: 
... must add a `product-summary` as a block .... 
or
... must add a product summary as a block ....

File: README.md (hunk 7 above)
Comment (vitoria) [reply README.md]: I think it's great! :100:

File: README.md [hunk 9]
```diff
@@ -17,31 +16,43 @@ See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for mor
...
   - [Styles API](#styles-api)
 - [Troubleshooting](#troubleshooting)
 
 ## Usage
-:construction: :construction: :construction:
 
-To use this app you need to import in your dependencies on `manifest.json`.
+This app uses our store builder with the blocks architecture. To know more about Store Builder [click here.](https://help.vtex.com/en/tutorial/understanding-storebuilder-and-stylesbuilder#structuring-and-configuring-our-store-with-object-object)
+
+To use this app, you need to import in your dependencies on `manifest.json`.
```
Comment (vitoriaHeliane) [inline README.md]: :hammer:  ... import in your `dependencies` on the `manifest.json` file.

Comment (vitoriaHeliane) [reply README.md]: :hammer:  To use this app, you need to add it in the `dependencies` on `manifest.json` file.

File: README.md (hunk 4 above)
Comment (lucis) [reply README.md]: @GustavoMA isn't better?
The minicart has one required block, that is `product-summary`.

File: README.md (hunk 7 above)
Comment (lucis) [reply README.md]: which rules*

File: README.md (hunk 4 above)
Comment (brunojdo) [reply README.md]: oh man

File: README.md (hunk 6 above)
Comment (brunojdo) [reply README.md]: this is a nit, I prefer the suggestion of Grammarly.

File: README.md (hunk 8 above)
Comment (brunojdo) [reply README.md:52]: 🤷‍♀️

**Review by GustavoMA — APPROVED** [review]

**Review by vitoria — APPROVED** [review]

**Pull request merged**